FatFs_SPI/sd_driver/spi.c

```diff
@@ -18,14 +18,30 @@
 //
 #include "spi.h"
 
+bool irqChannel1 = false;
+bool irqShared = true;

```

Comment:
Is there any reason these two variables couldn't be static?

Reply:
They probably should be; I just forgot to mark them as such.